sandbox/grist/table.py

```diff
@@ -79,13 +79,18 @@ def RecordSet(self):
   # and we decided camelCase was a more user-friendly choice for user-facing functions.
   def lookupRecords(self, **field_value_pairs):
     """
-    Returns the Records from this table that match the given field=value arguments. If
-    `sort_by=field` is given, sort the results by that field.
+    Name: lookupRecords
+    Usage: UserTable.__lookupRecords__(Field_In_Lookup_Table=value, ...)
+    Returns a RecordSet matching the given field=value arguments. The value may be any expression, 
+    most commonly a field in the current row (e.g. `$SomeField`) or a constant (e.g. a quoted string 
+    like `"Some Value"`) (examples below). 
+    If `sort_by=field` is given, sort the results by that field.
 
     For example:
     ```
-    People.lookupRecords(Last_Name="Johnson", sort_by="First_Name")
-    People.lookupRecords(First_Name="George", Last_Name="Washington")
+    People.*lookupRecords*(Email=$Work_Email)
```

Comment:
Ugh, sorry I didn't think of it until I tested, but you can't make text bold within a code block -- it would actually insert asterisks. The good news is that if you remove it, the syntax highlighting will just happen to highlight "lookupRecords" (a complete accident, due to the syntax highlighter guessing that the code is some obscure language), but it looks great :)